src/context.cpp

```diff
@@ -55,31 +92,57 @@ rs_context_base::rs_context_base()
 {
     context = rsimpl::uvc::create_context();
 
-    for(auto device : query_devices(context))
+    try
+    {
+        create_devices();
+    }
+    catch(std::shared_ptr<device> device)
     {
-        LOG_INFO("UVC device detected with VID = 0x" << std::hex << get_vendor_id(*device) << " PID = 0x" << get_product_id(*device));
+        LOG_WARNING("Failed to read camera info - cycling hub power");
 
-        if (get_vendor_id(*device) != VID_INTEL_CAMERA)
-            continue;
+        set_hub_power(*device, false, 1500);
```

Comment:
Are these values defined somewhere, maybe post a link to the hub api or #define them?